lib/pool.py

```diff
@@ -156,12 +160,18 @@ def restore_uefi_certs(self):
             self.saved_uefi_certs = None
 
     def clear_uefi_certs(self):
+        assert self.master.xcp_version < version.parse("8.3"), "function only relevant on XCP-ng 8.2"
         logging.info('Clearing pool UEFI certificates in XAPI and on hosts disks')
         self.master.ssh(['secureboot-certs', 'clear'])
         # remove files on each host
         for host in self.hosts:
             host.ssh(['rm', '-f', f'{host.varstore_dir()}/*'])
 
+    def clear_custom_uefi_certs(self):
```

Comment:
Adding docstrings to those funcs could help understand the difference with `clear_uefi_certs`